### PR TITLE
Translate doc distribute-credentials-secure into Chinese

### PR DESCRIPTION
**Pull request by lichuqiang**
CN doc translation

/cc @tengqm @zhangxiaoyu-zidif

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6292)

### CODE REVIEW

File: cn/docs/tasks/inject-data-application/distribute-credentials-secure.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+title: Distribute Credentials Securely Using Secrets
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/distribute-credentials-secure.md]: translate this

File: cn/docs/tasks/inject-data-application/distribute-credentials-secure.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    echo -n '39528$vdg7Jb' | base64
+
+结果显示 base-64 形式的用户名为 `bXktYXBw`，
+base-64 形式的密码为 `Mzk1MjgkdmRnN0pi`。
+
+## 创建 Secret
+
+这里是一个配置文件，可以用来创建存有用户名和密码的 Secret:
+
+{% include code.html language="yaml" file="secret.yaml" ghlink="/docs/tasks/inject-data-application/secret.yaml" %}
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/distribute-credentials-secure.md]: you may have to replicate the yaml file for the moment to make CI happy.
I hope this duplication issue can be solved in a decent way in future, then we can remove the duplicates.

File: cn/docs/tasks/inject-data-application/distribute-credentials-secure.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        Data
+        ====
+        password:   13 bytes
+        username:   7 bytes
+
+## 创建可以通过卷访问 secret 数据的 Pod
+
+这里是一个可以用来创建 pod 的配置文件：
+
+{% include code.html language="yaml" file="secret-pod.yaml" ghlink="/docs/tasks/inject-data-application/secret-pod.yaml" %}
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/distribute-credentials-secure.md]: same here.

**Comment by lichuqiang** [comment]
@tengqm done. Thanks

**Pull request merged**